src/components/commands-panel.tsx

```diff
@@ -142,31 +145,48 @@ export const CommandsPanel = React.memo((props: CommandsPanelProps) => {
       dropOffBehavior,
     );
   };
-
+  // If we don't have a configuration file with robots and places we should not render the commands forms because we will not be able to execute those commands.
   return (
     <React.Fragment>
-      <Accordion data-component="LoopForm">
-        <AccordionSummary
-          classes={{ content: classes.accordionSummaryContent }}
-          expandIcon={<ExpandMoreIcon />}
-        >
-          <Typography variant="h5">Loop Request</Typography>
-        </AccordionSummary>
-        <AccordionDetails className={classes.accordionDetail}>
-          <LoopForm requestLoop={handleRequestLoop} fleetNames={allFleets} />
-        </AccordionDetails>
-      </Accordion>
-      <Accordion data-component="DeliveryForm">
-        <AccordionSummary
-          classes={{ content: classes.accordionSummaryContent }}
-          expandIcon={<ExpandMoreIcon />}
-        >
-          <Typography variant="h5">Delivery Request</Typography>
-        </AccordionSummary>
-        <AccordionDetails className={classes.accordionDetail}>
-          <RobotDeliveryForm requestDelivery={handleDeliveryRequest} fleetNames={allFleets} />
-        </AccordionDetails>
-      </Accordion>
+      {!!resourcesContext.robots ? (
+        <>
+          <Accordion data-component="LoopForm">
+            <AccordionSummary
+              classes={{ content: classes.accordionSummaryContent }}
+              expandIcon={<ExpandMoreIcon />}
+            >
+              <Typography variant="h5">Loop Request</Typography>
+            </AccordionSummary>
+            <AccordionDetails className={classes.accordionDetail}>
+              <LoopForm
+                requestLoop={handleRequestLoop}
+                fleetNames={allFleets}
+                robotHandler={resourcesContext.robots}
+              />
+            </AccordionDetails>
+          </Accordion>
+          <Accordion data-component="DeliveryForm">
+            <AccordionSummary
+              classes={{ content: classes.accordionSummaryContent }}
+              expandIcon={<ExpandMoreIcon />}
+            >
+              <Typography variant="h5">Delivery Request</Typography>
+            </AccordionSummary>
+            <AccordionDetails className={classes.accordionDetail}>
+              <RobotDeliveryForm
+                requestDelivery={handleDeliveryRequest}
+                fleetNames={allFleets}
+                robotHandler={resourcesContext.robots}
+              />
+            </AccordionDetails>
+          </Accordion>
+        </>
+      ) : (
+        <Typography className={classes.errorText}>
+          You do not have a configuration file loaded, therefore the commands cannot be executed. To
```

Comment:
This error message may be confusing for normal users. I suggest something simpler like `There is an error while loading the commands panel (unable to load places metadata)`.

Reply:
Sounds good!